src/components/game/GameDetails.js

```diff
@@ -74,7 +74,7 @@ export const GameDetails = (props) => {
                     
                 })
         }
-
+        <button onClick={() => {props.history.push({ pathname: `upload-image/${games.id}` })}}>Upload Image</button>
```

Comment:
```suggestion
        <button onClick={() => {props.history.push({ pathname: `/upload-image/${games.id}` })}}>Upload Image</button>
```
There needs to be a slash in front up upload-image. Without it, react just appends `upload-image/:id` to the end of the url